I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/prologue.rpy", line 46, in script
    $ renpy.pause(1.0)
  File "renpy/common/00library.rpy", line 132, in extend
    renpy.exports.say(who, what, interact=interact)
IOError: Couldn't find file './sound/knocking_2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/prologue.rpy", line 46, in script
    $ renpy.pause(1.0)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/ast.py", line 613, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/exports.py", line 1142, in say
    who(what, interact=interact)
  File "renpy/common/00library.rpy", line 132, in extend
    renpy.exports.say(who, what, interact=interact)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/exports.py", line 1142, in say
    who(what, interact=interact)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/character.py", line 876, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/character.py", line 715, in do_display
    **display_args)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/character.py", line 507, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/display/core.py", line 2492, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/display/core.py", line 2791, in interact_core
    renpy.audio.audio.interact()
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/audio/audio.py", line 930, in interact
    periodic()
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/audio/audio.py", line 864, in periodic
    c.periodic()
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/audio/audio.py", line 403, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/audio/audio.py", line 67, in load
    rv = renpy.loader.load(fn)
  File "/Users/enikfeirgraw/Applications/renpy-6.99.11-sdk/renpy/loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file './sound/knocking_2'.

Darwin-16.4.0-x86_64-i386-64bit
Ren'Py 6.99.12.2.2029
WIP 0.01
